### ISSUE TITLE
Full command for wan 2.2 i2v dual noise

### ISSUE THREAD

**Issue by BenDes21** [kohya-ss/musubi-tuner#561]
Hi there, not gonna lie Im little bit lost about the command to launch for start a i2v training wan 2.2 on both high and low noise models with videos as datasets

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 src/musubi_tuner/wan_train_network.py \
    --task i2v-A14B \
    --dit {path to low noise} \
    --dit_high_noise {path to high noise} \
    --timestep_boundary 0.9 \
    --discrete_flow_shift 5.0 \
    --network_module networks.lora_wan \
    --dit path/to/wan2.1_xxx_bf16.safetensors \
    --dataset_config path/to/toml --sdpa --mixed_precision bf16 --fp8_base \
    --optimizer_type adamw8bit --learning_rate 2e-4 --gradient_checkpointing \
    --max_data_loader_n_workers 2 --persistent_data_loader_workers \
    --network_module networks.lora_wan --network_dim 32 \
    --timestep_sampling shift --discrete_flow_shift 3.0 \
    --max_train_epochs 16 --save_every_n_epochs 1 --seed 42 \
    --output_dir path/to/output_dir --output_name name-of-lora

this is the base but then  Im not sure about --min_timestep , --max_timestep and if more settings are needed for videos as data, if someone can help me its would be awesome. Thanks !

**Comment by kohya-ss** [comment]
When training both HIGH/LOW models, the timestep must be set to the full range, so `--min_timestep` and `--max_timestep` can be left at their default values, i.e., not specified.

**Comment by BenDes21** [comment]
> When training both HIGH/LOW models, the timestep must be set to the full range, so `--min_timestep` and `--max_timestep` can be left at their default values, i.e., not specified.

Thanks a lot! Will train with 96gb vram, do you see any optimisation to do with my config ?

**Comment by kohya-ss** [comment]
If you have 96GB of VRAM, the memory optimization option `--gradient_checkpointing` should be enough. If there is free VRAM, you can increase the batch size to relatively increase the training speed.
Check the discussion for more tips: https://github.com/kohya-ss/musubi-tuner/discussions

**Comment by BenDes21** [comment]
> If you have 96GB of VRAM, the memory optimization option `--gradient_checkpointing` should be enough. If there is free VRAM, you can increase the batch size to relatively increase the training speed. Check the discussion for more tips: https://github.com/kohya-ss/musubi-tuner/discussions
Thanks
